.docs/user-guide/config.md

```diff
@@ -256,3 +256,22 @@ supported:
 Docker   | docker
 
 The volume driver `docker` is automatically activated.
+
+### VOLUME MOUNT PRE-EMPTION
+There is a capability to pre-emptively detach any existing attachments to other
+instances before attempting a mount.  This will enable use cases for
+availability where another instance must be able to take control of a volume
+without the current owner instance being involved.  The operation is considered
+equivalent to a power off of the existing instance for the device.
+
+Property Name | Environment Variable | CLI Flag
+--------------|----------------------|-------------
+`volume.mountpreempt`   | `REXRAY_VOLUME_MOUNTPREEMPT`          | `--preempt`
```

Comment:
So I was going back and forth about the naming of the param, and hence ended up in something that doesn't make sense =)

Two things 

1) I see the param being used from the CLI in one way now, with the mount command.  Does it make sense to add on `rexray` to that? ie. `rexray mount --volumename=test --preempt`.  I could see it as `--volumepreempt` as well though.

2) I went back and forth about where to place it in the config file.  At one point I was considering it a driver level configurable item.  But it is now implemented at the vdm level, so probably makes sense the way you list it as a RR context setting for volumes.